src/main/kotlin/com/bugsnag/android/gradle/BugsnagPluginExtension.kt

```diff
@@ -50,7 +50,7 @@ open class BugsnagPluginExtension(objects: ObjectFactory) {
         .convention(false)
 
     val retryCount: Property<Int> = objects.property<Int>()
-        .convention(0)
+        .convention(1)
```

Comment:
I think this should stay as 0 because 0 should mean "attempt the request once, but don't try again if it fails". From reading the code it seems like `retryCount` has become `maxRequestCount`, which is a little bit different